docs/references/calfits_memo.tex

```diff
@@ -60,8 +60,8 @@ \section{Primary Header}\label{primary}
 	\item Gain-type: this axis stores the concatenation of the UVCal arrays [real(gain\_array), imaginary(gain\_array), flag\_array, input\_flag\_array, quality\_array], in that order.
 	If there is no input\_flag\_array or no quality\_array for this calibration, they are \emph{not} appended to the axis.
 	Correspondingly, in a ``gain''-type calibration, this axis may be of length 3 to 5, depending on the inclusion of the input\_flag\_array and quality\_array.
-	In older files, the quality\_array is always present while the input\_flag\_array was optional. In newer files, the input\_flag\_array is generally
-	not present while the quality\_array can be present or absent and a boolean header item \textbf{HASQLTY} will be present to indicate its presence
+	In older files, the quality\_array is always present while the input\_flag\_array was optional. In newer files, the input\_flag\_array has been deprecated (as of \texttt{pyuvdata} version 3.0)
+    while the quality\_array can be present or absent and a boolean header item \textbf{HASQLTY} will be present to indicate its presence
```

Comment:
I think this is almost there, it just reads a little awkwardly. Maybe:

```
In older files, the quality\_array is always present while the input\_flag\_array was optional. In newer files, the input\_flag\_array is not present (support for it in UVCal was deprecated as of \texttt{pyuvdata} version 3.0)
```

Reply:
Okay, updated in ab5e623ad23c9d8bca5a68a9f48d01be4d5cddfc.